### ISSUE TITLE
Create EBS Volume from snapshot at boot

### ISSUE THREAD

**Issue by ashelley** [aws/opsworks-cookbooks#147]
Hello,

I've been trying to use the aws_ebs_volume resource to create a volume at boot.  https://github.com/opscode-cookbooks/aws however I am having a problem doing this with this cookbook.... see https://github.com/opscode-cookbooks/aws/issues/69

I would like to create/attach an ebs volume at boot to an opsworks instance from an existing snapshot however I don't think the recipes built into opsworks support this.  

Basically there is no way to have an attached volume creating from an existing snapshot in opsworks.  Is there any good way to work around this issue?

**Comment by awsrequena** [comment]
Hello,

we don't support this cookbooks, please contact the aws-cookbook maintainers.  As I see you've also got some feedback to you issue https://github.com/opscode-cookbooks/aws/issues/69

"Looks like an issue with the right_aws gem? I am now using the fog gem and works fine." from @zivagolee .  Maybe that is helpful.

Good luck
